programmingPractice/programmingPractice/Functions.swift

```diff
@@ -111,3 +111,12 @@ func primeNumberChecker(value: Int) -> String {
     }
 
 }
+
+func linearSearch(arrayToSearch: [Int], value: Int) -> String {
+    //count use a ternary here but best not I suppose
+    if arrayToSearch.contains(value) {
```

Comment:
for i in array {
    if i = value {
        return("The array contains " + String(value)")
    }
}
return("The array does not contain " + String(value)")

Reply:
do that as extension task 💃 🗡 😢

Reply:
🏩 💌 💘 ❤️ 😍 😻 
🍪 🍪 🍪 🍪 🍪 🍪 
🍪 🍪 🍪 🍪 🍪 🍪 
🍪 🍪 🍪 🍪 🍪 🍪